tests/eth_general.py

```diff
@@ -860,3 +862,44 @@ def test_run_solc_with_remappings(self):
                 self.assertIn("lib/B.sol", source_list)
         finally:
             shutil.rmtree(d)
+
+    def test_jmpdest_check(self):
+        '''
+        '''    
```

Comment:
Can we fill the docstring explaining the testcase or at least linking to something ( ? )